src/VirtoCommerce.InventoryModule.Web/Module.cs

```diff
@@ -100,13 +101,18 @@ public void PostInitialize(IApplicationBuilder appBuilder)
             var mvcJsonOptions = appBuilder.ApplicationServices.GetService<IOptions<MvcJsonOptions>>();
             mvcJsonOptions.Value.SerializerSettings.Converters.Add(new PolymorphicInventoryJsonConverter());
 
+            var settingsManager = appBuilder.ApplicationServices.GetRequiredService<ISettingsManager>();
             var inProcessBus = appBuilder.ApplicationServices.GetService<IHandlerRegistrar>();
             inProcessBus.RegisterHandler<InventoryChangedEvent>(async (message, token) => await appBuilder.ApplicationServices.GetService<LogChangesChangedEventHandler>().Handle(message));
-            inProcessBus.RegisterHandler<InventoryChangedEvent>(async (message, token) => await appBuilder.ApplicationServices.GetService<IndexInventoryChangedEventHandler>().Handle(message));
+            if (settingsManager.GetValue("Inventory.Search.EventBasedIndexation.Enable", false))
```

Comment:
Use ModuleConstants instead of string constant.

Reply:
It is done.